rastervision/data/label_source/chip_classification_geojson_source_config.py

```diff
@@ -86,6 +86,7 @@ def validate(self):
             raise rv.ConfigError(
                 'You must set the uri for ChipClassificationGeoJSONSourceConfig'
                 ' Use "with_uri".')
+        check_uri_type(self.config.get('uri'))
 
```

Comment:
There are some changes that seemed to have happened underneath this PR branch.  This line in particular changes somewhat (see [here](https://github.com/jamesmcclain/raster-vision/commit/780c84f91b1857d5637fd3e9d991c628f016fe97)).